tests/test_scrump.py

```diff
@@ -194,9 +299,9 @@ def test_scrump_self_join_full(T):
 
 
 @pytest.mark.skip(reason="naive PRESCRUMP is not yet implemented")
```

Comment:
Do we have a plan for how to complete this? What else is needed in order to get our test coverage back to 100%?

Reply:
The coverage is not at 100% because of PRESCRIMP, and I haven't changed anything there, so I can't say...

For this test we need to implement a naive prescrimp to have a naive scrimp_plus_plus available, i.e. to be able to compute only a certain number of diagonals but with preprocessing included.

Reply:
Okay, we'll need to get the `naive_prescrimp` tests in so that our tests are back up to 100% before we can release v1.4.0. Let me know if you need help with that

Reply:
Yes it would be great if you could help, since I don't know how prescrimp works and don't have to much time at the moment...

Reply:
Cool. Then I can merge this and work on pre-scrimp